product_pricelist_supplierinfo/tests/test_product_supplierinfo.py

```diff
@@ -106,7 +106,60 @@ def test_pricelist_min_quantity(self):
         self.assertAlmostEqual(
             self.pricelist.get_product_price(self.product, 10, False), 50,
         )
+        # As the sequence of both supplieinfos is the same, the next field to order
```

Comment:
This also happens below

```suggestion
        # As the sequence of both supplierinfos is the same, the next field to order
```

Reply:
Done